src/api/.rubocop_todo.yml

```diff
@@ -373,6 +373,7 @@ Metrics/CyclomaticComplexity:
     - 'app/models/update_notification_events.rb'
     - 'app/models/user.rb'
     - 'app/models/user_ldap_strategy.rb'
+    - 'app/models/workflow_run_filter.rb'
```

Comment:
Hm, ignoring that I don't really like... Why is that so?

Reply:
Because that class is not that complex to understand as rubocop says

Reply:
I agree, it's a simple list of single conditions for the parameters.

Reply:
Then please create follow up cards. We are supposed to have a plan how to pay back tech-debt we take on.

Reply:
https://trello.com/c/W2QqLX6H